core/jazz_usermanagement/index.js

```diff
@@ -63,34 +62,34 @@ module.exports.handler = (event, context, cb) => {
 		const cognito = new AWS.CognitoIdentityServiceProvider({ apiVersion: '2016-04-19', region: config.REGION });
 
 		if (subPath.indexOf('reset') > -1) {
-			logger.info('User password reset Request::' + JSON.stringify(service_data));
-
-			validateResetParams(service_data)
-				.then(() => forgotPassword(cognito, config, service_data))
-				.then(() => function (result) {
-					logger.info("Password reset was successful for user: " + service_data.email);
-					return cb(null, responseObj({ result: "success", errorCode: "0", message: "Password reset was successful for user: " + service_data.email }));
-				})
-				.catch(function (err) {
+			logger.info('User password reset Request:' + JSON.stringify(service_data));
+
+			exportable.validateResetParams(service_data)
+			.then(() => {
+				return exportable.forgotPassword(cognito, config, service_data) })
+				.then(result => {
+						  logger.info("Password reset was successful for user: " + service_data.email);
+						  return cb(null, responseObj({ result: "success", errorCode: "0", message: "Password reset was successful for user: " + service_data.email }));
+					  })
+				.catch((err) => {
 					logger.error("Failed while resetting user password: " + JSON.stringify(err));
-
+					
 					if (err.errorType) {
 						// error has already been handled and processed for API gateway
 						return cb(JSON.stringify(err));
 					} else {
 						if (err.code) {
 							return cb(JSON.stringify(errorHandler.throwInputValidationError(err.code, err.message)));
 						}
-
 						return cb(JSON.stringify(errorHandler.throwInternalServerError("106", "Failed while resetting user password for: " + service_data.email)));
 					}
 				});
 		} else if (subPath.indexOf('updatepwd') > -1) {
 			logger.info('User password update Request::' + JSON.stringify(service_data));
 
-			validateUpdatePasswordParams(service_data)
-				.then(() => updatePassword(cognito, config, service_data))
-				.then(() => function (result) {
+			exportable.validateUpdatePasswordParams(service_data)
+				.then(() => { return exportable.updatePassword(cognito, config, service_data)})
```

Comment:
you dont need {return....} here. in ES6 you can just write the statement if its one line only. please update here and other places